File: node/createTable.js
'use strict'
const AWS = require('aws-sdk')
const config = require('../config/config')

const dynamoDB = new AWS.DynamoDB({
    region: 'us-east-2',
    accessKeyId: config.id,
    secretAccessKey: config.key
})

const tableConfig = {
    TableName: 'Example Table',
    KeySchema: [
        {AttributeName: 'id', KeyType: 'HASH'},
        {AttributeName: 'fact', KeyType: 'RANGE'}
    ],
    AttributeDefinitions: [
        {AttributeName: 'id', AttributeType: 'N'},
        {AttributeName: 'fact', AttributeType: 'S'}
    ],
    ProvisionedThroughput: {
        ReadCapacityUnits: 1,
        WriteCapacityUnits: 3
    }
}

dynamoDB.createTable(tableConfig, (error, data) => {
    error
        ? console.error('Unable to create table. Error JSON:', JSON.stringify(error, null, 4))
        : console.log('Created table. Table description JSON:', JSON.stringify(data, null, 4))
})

File: python/createTable.py
import boto3
import json

with open('./config/config.json', 'r') as file:
    config = json.loads(file.read())

dynamodb = boto3.resource(
    'dynamodb',
    region_name           = 'us-east-2',
    aws_access_key_id     = config['ID'],
    aws_secret_access_key = config['KEY']
)

response = dynamodb.create_table(
    AttributeDefinitions = [
        {
            'AttributeName': 'id',
            'AttributeType': 'N'
        },
        {
            'AttributeName': 'fact',
            'AttributeType': 'S'
        },
    ],
    TableName = 'ExampleTableTwo',
    KeySchema = [
        {
            'AttributeName': 'id',
            'KeyType': 'HASH'
        },
        {
            'AttributeName': 'fact',
            'KeyType': 'RANGE'
        }
    ],
    ProvisionedThroughput = {
        'ReadCapacityUnits': 1,
        'WriteCapacityUnits': 1
    }
)

print(response.table_status)